uk_bin_collection/uk_bin_collection/councils/GatesheadCouncil.py

```
@@ -97,32 +99,33 @@ def parse_data(self, page: str, **kwargs) -> dict:
             # Wait for any content that indicates results are loaded
             try:
                 WebDriverWait(driver, 15).until(
-                    EC.presence_of_element_located((By.XPATH, "//*[contains(text(), 'collection') or contains(text(), 'Collection') or contains(text(), 'bin') or contains(text(), 'Bin') or contains(text(), 'refuse') or contains(text(), 'Refuse') or contains(text(), 'recycling') or contains(text(), 'Recycling')]"))
+                    EC.presence_of_element_located(
+                        (
+                            By.XPATH,
+                            "//*[contains(text(), 'collection') or contains(text(), 'Collection') or contains(text(), 'bin') or contains(text(), 'Bin') or contains(text(), 'refuse') or contains(text(), 'Refuse') or contains(text(), 'recycling') or contains(text(), 'Recycling')]",
+                        )
+                    )
                 )
             except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)